.github/ci-files/composer.json

```diff
@@ -69,7 +69,7 @@
   "repositories": [
     {
       "type": "git",
-      "url": "https://github.com/dennisameling/FOSOAuthServerBundle.git"
+      "url": "https://github.com/mollux/FOSOAuthServerBundle.git"
```

Comment:
❓ I think it would be better if we still have to maintain this fork, that it were on a Mautic org repo rather than an individuals. Just thinking out loud, but then we can ensure that it is subject to our own security processes/branch protections and the like.

Reply:
I think that would be preferable nowadays if we are still having to support the fork - from the supply chain perspective I think we should be a bit tighter on this kind of thing just as a belt and braces approach!

Reply:
agreed! can you create a repo for this and give me access? I can move the code

Reply:
https://github.com/mautic/FOSOAuthServerBundle voila (I just forked it direct from the main repo)

Reply:
@RCheesley  I switched to that repo

Reply:
OK great - will just need to bring over any of our changes to that repo as well then ;)

Reply:
I added them in https://github.com/mautic/mautic/pull/11217/commits/607876fa220d5e49c1fa1d24871357276ab79f1f

Reply:
Sorry, I mean bring over our changes to https://github.com/mautic/FOSOAuthServerBundle from Dennis' fork, as the fork I made was from the original repo so it won't have our changes applied, I am guessing. Or if that is a problem maybe we can just ask Dennis if we can move the forked repo into our org.

Reply:
Those changes are present, see https://github.com/mautic/FOSOAuthServerBundle/tree/doctrine-fix
I pushed the exact same branch from Denis' fork + my php 8 compatibility change, so we're good in this case.

Reply:
Ahhh perfect, sorry I was not looking at the branch! 🤦